flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/sink/shuffle/DataStatisticsOperator.java

```diff
@@ -97,8 +97,15 @@ public void open() throws Exception {
   }
 
   @Override
+  @SuppressWarnings("unchecked")
   public void handleOperatorEvent(OperatorEvent evt) {
```

Comment:
missed from last PR. Iceberg coding style tries to avoid unnecessary abbreviations like `evt`. maybe just `event` or `operatorEvent`.